docs/override-docker-services.md

```diff
@@ -0,0 +1,66 @@
+## Overriding configs in a custom Ozone distribution
+
+ - Assume you've created a custom Ozone distribution folowing this [guide](https://docs.ozone-his.com/create-distro/)
+
+ - This will create a bare-bones Ozone implementation project that should look like this:
+ ```
+ ozone-gruzinia/
+├── README.md
+├── config
+│     └── openmrs
+│         ├── frontend_config
+│         └── initializer_config
+├── pom.xml
+├── readme
+│     └── impl-guide.md
+└── scripts
+    ├── mvnw
+    ├── mvnw.cmd
+    └── docker-compose-files.txt
+```
+ - Add the custom configs and make sure they're copied to the right directory.
+ - Add the build step in the `pom.xml` for the custom configs:
+
+ ```
+        <execution>
+            <!-- Copy over the Docker Compose override file -->
+            <id>Copy docker-compose-override.yml</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>
+                ${project.build.directory}/${project.artifactId}-${project.version}/run/docker/
+              </outputDirectory>
+              <overwrite>true</overwrite>
+              <resources>
+                <resource>
+                  <directory>${project.basedir}/scripts</directory>
+                  <includes>
+                    <include>docker-compose-override.yml</include>
+                  </includes>
+                  <filtering>true</filtering>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+ ```
```

Comment:
At first reading this section comes out as if the reader is already aware of what custom configs are and how to deal with them. Is it because this article is still supposed to be preceded by one about handling custom configs?

Reply:
This doc will be following the one that details how to [create your own Ozone distribution](https://docs.ozone-his.com/create-distro/), this assumes the reader is aware of custom configs.

Reply:
Mmm, no, this other page (ie. [create your own Ozone distribution](https://docs.ozone-his.com/create-distro/)) explains how to create your own distro (using the Maven archetype etc). It doesn't delve into the question of overriding configs within one's custom distro.

Overriding configs will certainly still deserve its own page because we will need to detail how this is done for each supported component, even if just for pointing to each component's docs in that regard.